homeassistant/components/google_assistant/trait.py

```diff
@@ -744,6 +744,12 @@ def sync_attributes(self):
             self.hass.config.units.temperature_unit
         )
 
+        # Add a default min max value for set temperature to display correctly in Google Assistant App
+        response["thermostatTemperatureRange"] = {
+            "minThresholdCelsius": 5.0,
+            "maxThresholdCelsius": 30.0,
+        }
+
```

Comment:
This will most probably not be accepted. Anyway I think it warrants its own PR, I don't really know much about the GA integration

Reply:
I removed the setting and added a direct override in deconz plugin for SPZB0001 devices.